### PR TITLE
mkswap, swap{on,off}: update page

### PR DESCRIPTION
**Pull request by acuteenvy**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages/linux/mkswap.md [hunk 1]
```diff
@@ -1,20 +1,17 @@
 # mkswap
 
-> Sets up a Linux swap area on a device or in a file.
+> Set up a Linux swap area on a device or in a file.
+> Note: `path/to/file` can either point to a regular file or a swap partition.
```
Comment (kbdharun) [inline pages/linux/mkswap.md:4]:
```suggestion
> Note: `{{path/to/file}}` can either point to a regular file or a swap partition.
```

File: pages/linux/swapoff.md [hunk 2]
```diff
@@ -1,20 +1,17 @@
 # swapoff
 
-> Disables device or file for swapping.
+> Disable devices and files for swapping.
+> Note: `path/to/file` can either point to a regular file or a swap partition.
```
Comment (kbdharun) [inline pages/linux/swapoff.md:4]:
```suggestion
> Note: `{{path/to/file}}` can either point to a regular file or a swap partition.
```

File: pages/linux/swapon.md [hunk 3]
```diff
@@ -1,24 +1,21 @@
 # swapon
 
-> Enables device or file for swapping.
+> Enable devices and files for swapping.
+> Note: `path/to/file` can either point to a regular file or a swap partition.
```
Comment (kbdharun) [inline pages/linux/swapon.md:4]:
```suggestion
> Note: `{{path/to/file}}` can either point to a regular file or a swap partition.
```

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution. I have some minor suggestions for this page and the PR is GTG.

File: pages/linux/mkswap.md (hunk 1 above)
Comment (acuteenvy) [reply pages/linux/mkswap.md:4]: Why put placeholders in the description? Does any client highlight them?

Comment (kbdharun) [reply pages/linux/mkswap.md:4]: Just now checked it, seems like it's not the case, I have seen pages highlighting placeholders too so anything is fine IG, will mark this as resolved.

**Review by mfrw — APPROVED** [review]
LGTM.
Thanks @acuteenvy

**Pull request merged**